lib/v2/swjtu/radar.js

```diff
@@ -18,4 +21,26 @@ module.exports = {
             },
         ],
     },
+    'jwc.swjtu.edu.cn': {
+        _name: '西南交通大学教务网',
+        dean: [
```

Comment:
Top-level object key can only match the domain name without the hostname.

Use
```js
'swjtu.edu.cn':{
    _name: '...',
    'dean.jwc': [
```
to match `dean.jwc.swjtu.edu.cn`